chatbot: make bootc: add message

vrothberg opened the pull request:
Add a message to the `make bootc` target to guide the user into creating a disk image, and by pointing to BIB and the Desktop extension.

@rhatdan @sallyom PTAL

lmilbaum reviewed (approved):
LGTM

rhatdan commented on recipes/natural_language_processing/chatbot/Makefile:
```diff
@@ -13,6 +13,12 @@ build:
 bootc: quadlet
 	podman build $${FROM:+--from $${FROM}} --cap-add SYS_ADMIN --build-arg "SSHPUBKEY=$(SSHPUBKEY)" -f bootc/Containerfile -t ${BOOTCIMAGE} .
+	@echo ""
+	@echo "Successfully built bootc image '${BOOTCIMAGE}'."
```
Should we also mention pushing to a registry and then installing via bootc install?

vrothberg replied:
Outside of my immediate scope but I am happy to add if you have text proposal.  Otherwise, we could move that into the README.

rhatdan replied:
@cgwalters WDYT?

vrothberg committed:
make bootc: add message

Add a message to the `make bootc` target to guide the user into creating
a disk image, and by pointing to BIB and the Desktop extension.

Signed-off-by: Valentin Rothberg <vrothberg@redhat.com>

vrothberg commented:
Rebased, @rhatdan PTAL

rhatdan commented:
LGTM
